package.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "mahi",
     "description": "Mahi client",
-    "version": "2.4.3",
-    "author": "MNX Cloud (mnx.io)",
+    "version": "2.4.4",
```

Comment:
This seems like a big enough feature to perhaps merit a `2.5.0` release?

Reply:
I agree, this change is a new feature for Mahi, so we should bump the version up.